.gitignore

```diff
@@ -1,7 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-/node_modules
+node_modules/
```

Comment:
this will ignore any folder with this name, regardless if it's in the same drecrtory of the .gitignore or not